src/assets/styles/article.css

```diff
@@ -0,0 +1,52 @@
+#article-body {
+	font-family: "Exo 2", sans-serif;
+	font-size: 18px;
+}
+
+@media screen and (min-width: 700px) {
```

Comment:
In case you decide to use `useStyles`, [this link](https://stackoverflow.com/questions/45847090/media-queries-in-material-ui-components) shows how to do media queries in MUI.